genesis/genesis-info.json.tmpl

```diff
@@ -422,9 +422,9 @@
                         "charge_symbol" : 91600047785729, "token_code" : "GOLOS", "charge_id" : 1,
                         "func" : {
                             "varssize" : 3,
-                            "operators" : [0, 0, 4],
-                            "values" : [{"kind" : 1, "idx" : 2}, {"kind": 0, "idx": 0}],
-                            "nums" : [1228800],
+                            "operators" : [0, 0, 3, 0, 4],
```

Comment:
maybe it's better to pre-calculate a constant?

Reply:
propose to let leaders do so later